packages/ppx_bsx/ppx_bsx.2.0.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+
+synopsis: "ReasonReact JSX for OCaml"
+description: """
+ReasonReact JSX v3 for OCaml, ReasonReact 0.7+ and BuckleScript 6.0+ required
+"""
+maintainer: "CHEN Xian-an <xianan.chen@gmail.com>"
+authors: "CHEN Xian-an <xianan.chen@gmail.com>"
+tags: [ "BuckleScript" "ReasonReact" "React" "JSX" ]
+license: "MIT"
+homepage: "https://github.com/cxa/ppx_bsx"
+dev-repo: "git+https://github.com/cxa/ppx_bsx.git"
+bug-reports: "https://github.com/cxa/ppx_bsx/issues"
+doc: "https://github.com/cxa/ppx_bsx"
+build: [
+  [ "dune" "subst" ] {pinned}
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+depends: [
+  "ocaml" { = "4.06.1" }
```

Comment:
is there a reason why you rely on this specific version of the compiler?

Reply:
Yes, BuckleScript on works on 4.02.3 and 4.06.1, but ppxlib doesn't support 4.02.3, so only 4.06.1 supported this time.

Reply:
but your package doesn't use reason. I'm not sure to understand

Reply:
`ppx_bsx` is a preprocessor, which will generate intermediate files, BuckleScript compiler (also OCaml compiler) will use those files to generate JS. So version matching is required. 

Just like you can't use 4.07.x to compile the files which preprocessed by 4.06.x.

Not a compiler expert, Hope my description can be understood.

Reply:
ok fair enough